### PR TITLE
jme3-plugins:  rename the TrackData.checkTimesConsistantcy() method

### PR DESCRIPTION
**Pull request by stephengold**
to address issue #1816

### CODE REVIEW

File: jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/TrackData.java [hunk 1]
```diff
@@ -165,19 +165,30 @@ public void update() {
...
+     * {@link #times}, {@link #translations}, {@link #rotations}, and
+     * {@link #scales} vectors all have the same length, if present.
+     *
+     * @throws AssetLoadException if the lengths differ
+     */
+    public void checkTimesConsistency() {
         if ((translations != null && times.length != translations.length)
                 || (rotations != null && times.length != rotations.length)
                 || (scales != null && times.length != scales.length)) {
             throw new AssetLoadException("Inconsistent animation sampling ");
```
Comment (capdevon) [inline jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/TrackData.java:184]: space at the end of the error message is needed?

Comment (stephengold) [reply jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/TrackData.java:184]: can be removed

Comment (stephengold) [reply jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/TrackData.java:184]: However, I think removal shouldn't be part of this PR.

**Comment by stephengold** [comment]
Unless there's more to discuss, I plan to integrate this PR in about 24 hours.

**Pull request merged**